lib/src/dashmon.dart

```diff
@@ -44,29 +46,33 @@ class Dashmon {
   }
 
   void _processLine(String line) {
```

Comment:
We can probably now get rid of this `_processLine` method, since it is simply calling `_print`. Wdyt?

Reply:
I think it's good, too. I just left the `_processError` function in case it needed other work like displaying different colors on the console.